x-pack/plugins/siem/public/components/field_renderers/field_renderers.tsx

```diff
@@ -137,14 +138,21 @@ interface DefaultFieldRendererProps {
   maxOverflow?: number;
 }
 
+// TODO: This causes breaks between elements until the ticket below is fixed
+// https://github.com/elastic/ingest-dev/issues/474
 export const DefaultFieldRenderer = pure<DefaultFieldRendererProps>(
   ({ rowItems, attrName, idPrefix, render, displayCount = 1, maxOverflow = 5 }) => {
     if (rowItems != null && rowItems.length > 0) {
       const draggables = rowItems.slice(0, displayCount).map((rowItem, index) => {
         const id = escapeDataProviderId(`${idPrefix}-${attrName}-${rowItem}`);
         return (
           <EuiFlexItem key={id} grow={false}>
-            {index !== 0 && <>,&nbsp;</>}
+            {index !== 0 && (
+              <>
+                {','}
+                <Spacer />
```

Comment:
NOTE: Spacer introduced and the `&nbsp;` removed since it did not pass the linter rule. Also, it is more problematic using it vs. using a more configurable spacer.